cartographer/mapping/internal/3d/local_trajectory_builder_3d.cc

```diff
@@ -307,38 +307,26 @@ LocalTrajectoryBuilder3D::AddAccumulatedRangeData(
     kLocalSlamInsertIntoSubmapFraction->Set(insert_into_submap_fraction);
   }
   const auto wall_time = std::chrono::steady_clock::now();
-  const double cpu_thread_time_sec = common::GetThreadCpuTimeSeconds();
+  const double cpu_thread_time_seconds = common::GetThreadCpuTimeSeconds();
   if (last_wall_time_.has_value()) {
-    const auto wall_time_duration =
-        wall_time - last_wall_time_.value();
+    const auto wall_time_duration = wall_time - last_wall_time_.value();
     kLocalSlamLatencyMetric->Set(common::ToSeconds(wall_time_duration));
     if (sensor_duration.has_value()) {
-      kLocalSlamRealTimeRateRatio->Set(
-          common::ToSeconds(sensor_duration.value()) / common::ToSeconds(wall_time_duration));
-      // TODO(spielawa): remove debug output
-      LOG(INFO) << "wall_time_duration: " << common::ToSeconds(wall_time_duration) << 
-      " --- RealTimeRateRatio: " <<
-      100 * common::ToSeconds(sensor_duration.value()) / common::ToSeconds(wall_time_duration)  << " %";
-    } else {
-      LOG(INFO) << "wall_time_duration: " << common::ToSeconds(wall_time_duration); 
+      kLocalSlamRealTimeRatio->Set(
+          common::ToSeconds(sensor_duration.value()) /
+          common::ToSeconds(wall_time_duration));
     }
   }
-  if (last_thread_cpu_time_sec_.has_value()) {
-      const double thread_cpu_duration = cpu_thread_time_sec - 
-        last_thread_cpu_time_sec_.value();
+  if (last_thread_cpu_time_seconds_.has_value()) {
+    const double e_seconds =
+        cpu_thread_time_seconds - last_thread_cpu_time_seconds_.value();
```

Comment:
and ultra-nit here: make both consistent, like (last_)thread_cpu_time_seconds or (last_)cpu_thread_time_seconds and cpu_thread_time.